db/dataset_eval.py

```diff
@@ -24,8 +24,13 @@
 # Filter types are defined in `eval_filter_type` type. See schema definition.
 FILTER_ARTIST = "artist"
 
+# Default values for parameters
+preprocessing = ['basic', 'lowlevel', 'nobands', 'normalized', 'gaussianized']
```

Comment:
these default values are repeated three times in this PR, once here, once in [forms.py](https://github.com/metabrainz/acousticbrainz-server/pull/257/files#diff-d426557fa42c5459f53956bcfdc1e0aaR18) and once in [views/datasets.py](https://github.com/metabrainz/acousticbrainz-server/pull/257/files#diff-b1772f3502d312f3d84deae191687459R19). In general, it is a good idea to avoid duplication of code and constants as much as possible. Please define these in `dataset_eval/__init__.py` and import them, reducing duplication. 

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself